### PR TITLE
GitHub doc update

### PR DESCRIPTION
**Pull request by SionHu**
1. OAuth instruction 
2. New api link
3. cvs_update file rename and introduction.

### CODE REVIEW

**Comment by mardi08** [comment]
@nhendy @SionHu @rg167 please review the documentation content update and put feedback on this pull requests.

File: docs/implementationDetail/GoogleFusionTable.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Google Fusion Table
 
-This page is created for documenting every feature relevant to Google Fusion Table
+
+This script is built to fecth cmaera APIs on [cam2-camera-api](https://purduecam2project.github.io/CameraDatabaseAPI/#api-cameras-camById).
```
Comment (nhendy) [inline docs/implementationDetail/GoogleFusionTable.md]: A typo.

**Comment by rg167** [comment]
The only thing I've noticed so far is a typo on line 5 of GoogleFusionTable.md ("This script is built to fecth cmaera APIs on...")

**Comment by mardi08** [comment]
for general documentation, avoid using first person. use third person instead. 

for example, 
avoid 
``` Mine reads "Sync Spreadsheet To Fusion Table."```

**Pull request merged**